src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Logging/ProjectOutputWindowProjectLogger.cs

```diff
@@ -33,6 +33,7 @@ public bool IsEnabled
         {
             get { return _options.IsProjectOutputPaneEnabled; }
```

Comment:
How do we feel about using expression bodied members for one-line getters like this?  Is there an existing policy about it?